dftimewolf/lib/logging_utils.py

```diff
@@ -0,0 +1,71 @@
+"""Module providing custom logging formatters."""
+import logging
+import random
+
+def _GenerateColorSequences():
+  """Generates ANSI codes for 256 colors."""
+  sequences = []
+  for i in range(0, 16):
```

Comment:
`for i ...` and `for j ....` this feels like a copy/paste from stack overflow... can we at least name these variables (style guide)

Reply:
I honestly don't know what to call them other than counters. This is how the numbers match to the colors: https://en.wikipedia.org/wiki/ANSI_escape_code#8-bit

Reply:
well, that's a good question ... let's leave it as is....